games-emulation/higan/higan-116_pre.ebuild

```diff
@@ -0,0 +1,100 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit desktop toolchain-funcs xdg
+
+DESCRIPTION="A multi-system emulator focused on accuracy, preservation, and configurability"
+HOMEPAGE="https://github.com/higan-emu/higan"
+SRC_URI="https://github.com/higan-emu/${PN}/archive/nightly.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="alsa ao +gtk openal +opengl oss +pulseaudio +sdl udev xv"
+
+RDEPEND="
+	x11-libs/libX11
+	x11-libs/libXext
+	alsa? ( media-libs/alsa-lib )
+	ao? ( media-libs/libao )
+	openal? ( media-libs/openal )
+	opengl? ( virtual/opengl )
+	pulseaudio? ( media-sound/pulseaudio )
+	sdl? ( media-libs/libsdl2[joystick] )
+	udev? ( virtual/udev )
+	xv? ( x11-libs/libXv )
+	gtk? (
+		x11-libs/cairo
+		x11-libs/gtk+:3
+		x11-libs/gtksourceview:3.0
+	)
+	!gtk? (
+		dev-qt/qtcore:5
+		dev-qt/qtgui:5
+		dev-qt/qtwidgets:5
+	)"
+DEPEND="${RDEPEND}"
+BDEPEND="virtual/pkgconfig"
+
+S="${WORKDIR}/${PN}-nightly"
+
+src_compile() {
+	local makeopts=(
+		platform=linux
+		compiler="$(tc-getCXX)"
+		hiro="$(usex gtk gtk3 qt5)"
+	)
+
+	local drivers=(
+		video.xshm
+		input.xlib
+		$(usex alsa audio.alsa "")
+		$(usex ao audio.ao "")
+		$(usex openal audio.openal "")
+		$(usex opengl "video.glx video.glx2" "")
+		$(usex oss audio.oss "")
+		$(usex pulseaudio "audio.pulseaudio audio.pulseaudiosimple" "")
+		$(usex sdl input.sdl "")
+		$(usex udev input.udev "")
+		$(usex xv video.xvideo "")
+	)
+
+	local coreopts=(
+		cores="cv fc gb gba md ms msx ngp pce sfc sg ws"
+		ruby="${drivers[@]}"
+		build=performance
+		local=false
+	)
+
+	# Make higan
+	emake "${makeopts[@]}" "${coreopts[@]}" -C higan-ui
+
+	# Make icarus
+	emake "${makeopts[@]}" -C icarus
+}
+
+src_install() {
+	# Install higan
+	dobin higan-ui/out/higan
```

Comment:
```
>>> Install games-emulation/higan-116_pre into /var/tmp/portage/games-emulation/higan-116_pre/image
!!! dobin: higan-ui/out/higan does not exist
```

Looks like it should be `dobin higan-ui/out/higan-ui` instead.

Reply:
No, this was renamed in https://github.com/higan-emu/higan/commit/a8d19935fac5a9ae2a0a2b00284af937da269d59

Is it trying to build an old version of the tarball?